include/TPP/Dialect/Xsmm/XsmmOps.td

```diff
@@ -226,4 +256,34 @@ def Xsmm_UnaryDispatchOp : Xsmm_Op<"unary.dispatch",[Pure]> {
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// MatmulDispatchOp
+//===----------------------------------------------------------------------===//
+
+// TODO: all the `flags` should be a list.
+class Xsmm_MatmulAndBrgemmDispatchOp<string mnemonic, list<Trait> traits = []> :
```

Comment:
I'd call it `GEMMLike`?

Reply:
Went for MatmulLike as the op is called Matmul